config-sidecar/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM node:14-stretch-slim
+
+WORKDIR /app
+
+RUN chmod g+w /app
+
+COPY package*.json /app/
+
+RUN npm i --only=production
+
+COPY script.mjs .
+
+# RUN useradd -ms /bin/bash user && usermod -a -G root user
+
+# USER user
```

Comment:
can it be removed?

Reply:
so add a note that it helps for local debugging

Reply:
added a note